x-pack/elastic-agent/_meta/config/common.reference.p2.yml.tmpl

```diff
@@ -119,6 +119,12 @@ inputs:
 #       host: localhost
 #       # Port on which the HTTP endpoint will bind. Default is 0 meaning feature is disabled.
 #       port: 6791
+#       # Metrics buffer config
+#       buffer:
+#           enabled: false
+#           period: 10s
+#           size: 60
+#           namespaces: ['stats']
```

Comment:
I've added the full buffer config here, but I'm not sure if we should. Currently the `enabled` flag is the only one that's passed (it's injected via command line args at the moment).

Reply:
I agree if its not exposed then it should be hidden.